tincr/io/device/xdlrc.tcl

```diff
@@ -189,10 +200,19 @@ proc ::tincr::write_xdlrc { args } {
     puts "Process ended at [clock format $end_time -format %H:%M:%S] on [clock format $end_time -format %D]"
 }
 
-proc ::tincr::write_xdlrc_tile { tile outfile brief } {
+proc ::tincr::write_xdlrc_tile { tile outfile brief is_series7 } {
```

Comment:
Just compute `is_series7` in the `write_xdlrc_tile` proc. No need to pollute the procedure's prototype.